### DISCUSSION TITLE
PKSIM Beginner, How to add first order Absorption Rate Constant?

### DISCUSSION THREAD

**Discussion by jasonargeseanu** [Open-Systems-Pharmacology/Forum#957] [Questions & Problems]
Hi, I'm a student who is learning how to use PKSIM. I am interested in taking data for a simple linear 1 compartment drug and modeling it in PKSIM. Where can I input a Ka value for the drug's oral absorption?

Thanks!

**Comment by prvmalik** [comment]
Hi,

Generally speaking this cannot be done. PK-Sim is a mechanistic modeling tool and therefore requires you to describe oral absorption within a mechanistic framework of the gastrointestinal tract (called the ACAT model).

For a BCS class 1 compound, try this to get the ball rolling:
Add formulation building block "Dissolved" and name it "Solution" (representative of administering an oral solution, or representative of administering an oral tablet that dissolves very quickly)
Add administration protocol and specify an Oral dose

Create a simulation, configure it to include your oral dose administration protocol and your new formulation building block called "solution."

Play around with the parameter "specific intestinal permeability" to see if you can describe your Cmax & Tmax correctly. Optimize using the parameter identification tool.

Alternatively you can describe the dissolution of an oral formulation with a weibull function or power function that approximates Ka
